podvm/addons/gpu/README.md

```diff
@@ -0,0 +1,75 @@
+## Introduction
+
+This addon enables nvidia GPU support in the podvm image.
+
+You need to specify the GPU instance types in the cloud-api-adaptor configMap (peer-pods-cm).
+
+Here is an example. Replace it as appropriate depending on the specific provider and region
+
+```
+# For AWS 
+PODVM_INSTANCE_TYPES: "t3.small,c5.xlarge,p3.2xlarge"
+
+# For Azure
+AZURE_INSTANCE_SIZES: "Standard_D8as_v5,Standard_D4as_v5,Standard_NC6s_v3,Standard_NC4as_T4_v3"
+
+```
+
+Example pod definition:
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  name: gpu-test
+  labels:
+    app: test
+  annotations:
+    io.katacontainers.config.hypervisor.machine_type: Standard_NC4as_T4_v3
```

Comment:
```suggestion
    io.katacontainers.config.hypervisor.machine_type: Standard_NC4as_T4_v3
    io.containerd.cri.runtime-handler: kata-remote
```

don't we need this?

Reply:
In my testing it was not needed. The annotation will be used by CAA to spin up the specific instance and the runtimeclass is part of of the pod spec itself.

Reply:
I think it will break once we switch to kata#main. AFAIU the nydus-snapshotter requires this.